README.md

```diff
@@ -19,6 +19,42 @@ You can use `stack build` to build `baseDAO` executable.
 
 Run `stack exec -- baseDAO --help` to see available commands.
 
+### Deploying a contract
+
+You can dump the entire contract code into a `BaseDao.tz` file using the following command:
+
+```sh
+stack exec baseDAO -- print -n baseDAO
+```
+
+For now this can only produce a contract with empty proposal metadata and the simplest possible configuration.
+
+The initial storage of the contract can be produced using
+
+
+```sh
+admin=<administrator address>
+stack exec baseDAO -- storage-BaseDAO --admin $admin > storage
+```
+
+Later the contract can be originated with `tezos-client`:
+
+```sh
+tezos-client originate contract baseDAO transferring 0 from alice running BaseDAO.tz --init "$(<storage)" --burn-cap 17.0
+```
+
+How to install `tezos-client` executable can be found at official [Tezos installation page](http://tezos.gitlab.io/introduction/howtoget.html) or in [our `tezos-packaging` repository](https://github.com/serokell/tezos-packaging).
+
+### Documentation
+
+You can obtain the actual version of the contract documentation via
+
+```sh
+stack exec baseDAO -- document -n BaseDAO
+```
+
+This command will dump the documentation to `BaseDAO.md` file.
```

Comment:
Above we say where one can open this documentation. I think an unprepared reader might be confused because it's not obvious what's the relation between documentation in this paragraph and aforementioned documentation. We know it's the same documentation, but an arbitrary person doesn't.